### PR TITLE
Temp redesign

### PR DESCRIPTION
**Pull request by jaison2000b**
image

### CODE REVIEW

**Comment by jaison2000b** [comment]
of course the image I got was when the Carousel was transitioning

**Comment by jaison2000b** [comment]
k, well don't care about css semantics, suck my dick

File: index.html [hunk 1]
```diff
@@ -84,7 +83,7 @@ <h3 class="fofbb display-4">MERCH!</h3>
                         </div>
                     </div>
 
-                    <div class="carousel-item">
+                    <div class="carousel-item active">
```
Comment (xtoferprime) [inline index.html]: the first item should be the active item on pageload

File: index.html [hunk 2]
```diff
@@ -68,7 +67,7 @@ <h1 class="display-3 f13"><span class="drop_shadow">THE BAWDY CASTE</span></h1>
 
                 <div class="carousel-inner" role="listbox">
 
-                    <div class="carousel-item active">
```
Comment (xtoferprime) [inline index.html]: the first item should be active on pageload

File: index.html [hunk 3]
```diff
@@ -112,53 +111,28 @@ <h3 class="fofbb display-4">SFPRIDE!</h3>
             <div class="row">
 
                 <div id="rocky-horror" class="col">
-                    <h2 class="df" style="margin-bottom: 0%;">Rocky Horror Picture Show</h2>
-                    <p class="fofbb" style="color:black;margin-top: 0;padding-top: 0;">The last Showing at the Clay!!<br> January 25th 2020!</p>
-                    
+
+                    <h2 class="df" style="margin-bottom: 0%;">The Rocky Horror Picture Show</h2><br>
```
Comment (xtoferprime) [inline index.html]: inline styling removes the bottom margin, and then a <br> is added? 

which one is it, no space at the bottom, or a line break?

File: index.html [hunk 4]
```diff
@@ -112,53 +111,28 @@ <h3 class="fofbb display-4">SFPRIDE!</h3>
...
+
                     <div class="col-text-center">
-                        <img src="/images/Clay.png" alt="Clay Theatre">
+                       <!-- <img src="/images/Balboa.png" alt="Balboa Theatre">
```
Comment (xtoferprime) [inline index.html]: this code exposes non-public information.

@crgrafton urgent rebase may be required

File: index.html [hunk 5]
```diff
@@ -112,53 +111,28 @@ <h3 class="fofbb display-4">SFPRIDE!</h3>
...
                         <h4 size="10px" class="fofbb" style="color:black;margin-top: 0;padding-top: 0;">San Francisco, CA</h4>
                         <p>
-                            <a class="btn btn-lg btn-danger" target="_blank" href="https://bit.ly/2tE50Nu">
+                            <a class="btn btn-lg btn-danger" target="_blank" href="">
```
Comment (xtoferprime) [inline index.html]: `.disable` buttons that do nothing, or users will be frustrated

File: index.html (hunk 2 above)
Comment (jaison2000b) [reply index.html]: was that something I changed?

File: index.html (hunk 3 above)
Comment (jaison2000b) [reply index.html]: I was given feedback for linebreak... the margin-bottom is stale code

File: index.html (hunk 4 above)
Comment (jaison2000b) [reply index.html]: boo-hoo

File: index.html (hunk 5 above)
Comment (jaison2000b) [reply index.html]: ? this is disabled

File: index.html (hunk 2 above)
Comment (xtoferprime) [reply index.html]: Yes... It was introduced by one of your commits...

**Comment by jaison2000b** [comment]
@crgrafton sh*t already takes too long to get approved and noone actually cares enough to do that including yourself

>  "his should really probably be at least two PRs, one to disable lebowski, and the other to reformat what's left."

>don't expose non-public information i

if someone is interested enough to look at the code of the site they deserve non-public information

File: index.html (hunk 2 above)
Comment (jaison2000b) [reply index.html]: this came about because at one point (to run code) It was dropped into a dev tools box and chrome automatically changed it to that

Comment (jaison2000b) [reply index.html]: changing it back

**Comment by jaison2000b** [comment]
> k, well don't care about css semantics, suck my dick

done

File: index.html (hunk 4 above)
Comment (jaison2000b) [reply index.html]: Not into it, don't caree

**Comment by jaison2000b** [comment]
@xtoferprime

File: index.html (hunk 4 above)
Comment (crgrafton) [reply index.html]: 🤷‍♂ 😞

**Comment by crgrafton** [comment]
image
image

**Comment by crgrafton** [comment]
image

**Pull request merged**